src/content/blog/astro-4130.mdx

```diff
@@ -0,0 +1,89 @@
+- [**Stable: Content Collections JSON Schemas**](#stable-content-collections-json-schemas)
+- [**Logging Improvements**](#logging-improvements)
+- [**Support for remote LibSQL databases**](#support-for-remote-libsql-databases)
+
+To upgrade an existing project, use the automated `@astrojs/upgrade` CLI tool. Alternatively, upgrade manually by running the upgrade command for your package manager:
+
+```sh
+# Recommended:
+npx @astrojs/upgrade
+
+# Manual:
+npm install astro@latest
+pnpm upgrade astro --latest
+yarn upgrade astro --latest
+```
+
+## Stable: Request Rewriting
+
+[In Astro 4.8](https://astro.build/blog/astro-480/#experimental-request-rewriting), we added experimental support for request rewriting. In Astro 4.13, we're excited to announce that request rewriting is now stable! This feature allows you to rewrite incoming requests to different paths or URLs, enabling you to create custom routing rules for your site.
+
+If you were using request rewriting in Astro 4.8, you can now remove the `experimental.rewrites` option from your Astro config. The feature is now enabled by default.
+
+```diff
+import { defineConfig } from 'astro';
+
+export default defineConfig({
+-  experimental: {
+-    rewriting: true
+-  }
+})
+```
+
+As a reminder, request rewriting is a powerful feature that can be used to render another page without changing the URL of the browser in Astro pages and endpoints. For more information on this feature, check out the [Astro documentation](https://docs.astro.build/en/guides/routing/#rewrite).
+
+## Stable: Content Collections JSON Schemas
+
+[In Astro 4.5](https://astro.build/blog/astro-450/#experimental-json-schemas-for-data-collections), we introduced the (experimental) ability to automatically generate JSON schemas for data content collections. In Astro 4.13, this feature is now stable! If you were using content collection JSON schemas in Astro 4.5, you can now remove the `experimental.contentCollectionJsonSchema` option from your Astro config. The feature is now enabled by default.
+
+```diff
+import { defineConfig } from 'astro';
+
+export default defineConfig({
+-  experimental: {
+-    contentCollectionJsonSchema: true
+-  }
+})
+```
+
+When you define a data content collection in your Astro project, Astro will now automatically generate a JSON schema for the collection. This schema can then be used in your editor to provide autocompletion, validation and more for your data. For more information on this feature, check out the [Astro documentation](https://docs.astro.build/en/guides/content-collections/#enabling-json-schema-generation).
+
+## Logging Improvements
+
+As part of our endless crusade against slow builds, Astro 4.13 includes a small improvement to the logging output during the build process. Pages that took a long time will now have their time logged in red, making it easier to spot bottlenecks in your build process. May there be less of these in your future!
```

Comment:
```suggestion
As part of our endless crusade against slow builds, Astro 4.13 includes a small improvement to the logging output during the build process. Pages that take a long time will now have their time logged in red, making it easier to spot bottlenecks in your build process. May there be fewer of these in your future!
```
Present tense feels a bit more natural here since the rest of the sentences are mostly present tense verbs?

I think an image of the output would go really nicely here!